### DISCUSSION TITLE
When switching languages, you need to refresh the page to take effect.

### DISCUSSION THREAD

**Discussion by eevaery** [nuxt-modules/i18n#3082] [Q&A]
![CleanShot 2024-09-02 at 12 43 44@2x](https://github.com/user-attachments/assets/14c01c97-7f07-4b84-a649-5a0c9906bcb5)
![CleanShot 2024-09-02 at 12 44 06@2x](https://github.com/user-attachments/assets/92d3bd9e-3c36-4c08-a62b-0a713a549a1d)

**Comment by BobbieGoede** [comment]
Please open an issue and provide a minimal reproduction 🙏

**Comment by eevaery** [comment]
Sorry, I made a mistake.
